app/components/rdf-form-fields/begrotingstabel-total-field.js

```diff
@@ -0,0 +1,168 @@
+import Component from '@glimmer/component';
+
+import { A } from '@ember/array';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { guidFor } from '@ember/object/internals';
+import { XSD, triplesForPath } from '@lblod/submission-form-helpers';
+import { restartableTask } from 'ember-concurrency';
+
+export default class BegrotingstabelTotalFieldComponent extends Component {
+  id = guidFor(this);
+
+  @tracked totals = A([]);
+  @tracked comparisonWarningMessage;
+  @tracked collectionErrorMessage;
+
+  constructor() {
+    super(...arguments);
+
+    this.store.registerObserver(() => {
+      // This will potentially run a lot of times. We can probably check the delta and see if the change affects this component, but due to time constraints we're skipping that for now.
+      // It's also possible that the delta checking is more intensive than the actual computations, but that would have to be benchmarked.
+      this.calculateTotals.perform();
+    }, this.id);
+  }
+
+  @action
+  formatAmount(amount) {
+    if (!amount) {
+      amount = 0;
+    }
+
+    return new Intl.NumberFormat('nl-BE', {
+      style: 'currency',
+      currency: 'EUR',
+    }).format(amount);
+  }
+
+  calculateTotals = restartableTask(async () => {
+    const collections = this.store
+      .match(this.field.uri, XSD('target'), undefined, this.graphs.formGraph)
+      .map((st) => st.object);
+
+    if (collections.length !== 2) {
+      this.collectionErrorMessage = `Tabel heeft te weing informatie. Voeg twee xsd:targets toe. bv. xsd:target ( ext:Expense ext:amount ), ( ext:Income ext:amount );`;
```

Comment:
Should we add a translation file for just the `BegrotingstabelTotal` component?

Reply:
These translations can than be easely added to the project that will be using this component.